best-practice-in-development/best-practice-in-client-side-development.md

```diff
@@ -0,0 +1,31 @@
+# Best Practices in Client-side Development
+
+### Projected Time
+About 20-30 minutes
+- Lesson: 20 min
+
+### Prerequisites
+[HTML](https://github.com/Techtonica/curriculum/blob/master/html/html.md) and [CSS](https://github.com/Techtonica/curriculum/blob/master/css/css.md)
+
+### Motivation
+- During development, we always focus on writing runnable code but we mostly ignore some of the best practices that must be followed during development which makes our code more readable & manageable when we are working in a team of developers.
+- The developer of new code and maintainer of existing code can communicate and coordinate easily if they adopt best practices.
+- Best practices are used to improve the quality of the software product.
+
+### Objectives
+**Participants will be able to:**
+- Understand the importance of dummy content during development.
+
+### Specific Things To Teach
+- Importance of Lorem ipsum text and images
+
+### Lesson
+
+- [About dummy text Lorem ipsum (10 min)](https://www.blindtextgenerator.com/about-lorem-ipsum) - Lorem Ipsum refers to text that developer and designer use as replacement text when the real text is not available. The purpose of Lorem Ipsum is to create a natural looking block of text (sentence, paragraph, page, etc.) that doesn't distract from the layout. Similar there is Lorem Picsum which refers to images that developer/designer use as filler images when the real images are not available.
```

Comment:
Well other things looks good to me, but it would be nice if you can add an example over here in this doc, to explain a bit about "Lorem ipsum".

Like: (add somethiing better, this is just an example)
Developer uses Lorem ipsum while making web apps,
They use it in generating placeholding content, etc.

PS: This will make it easier to understand this doc.

Reply:
@saberprashant I didn't get you what you want me to do in this review. 
can you elaborate on it?

Reply:
Yup sure, as I said, give an example over here,

Let's say if someone wants to create a website and while creating that website every developer makes a template first where they have to fill out some random content in it so that others can see how the website is looking with that content.
So, I am saying, to include this kind of example in the section.

Reply:
@saberprashant i.e. you want me to include guided practice section which contains an example where I can show how we can use dummy text.

Reply:
@saberprashant Done